mod.ts

```diff
@@ -36,8 +36,8 @@ export class Renderer extends Marked.Renderer {
     return `<h${level} id="${slug}"><a class="anchor" aria-hidden="true" tabindex="-1" href="#${slug}"><svg class="octicon octicon-link" viewBox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>${text}</h${level}>`;
   }
 
-  image(src: string, title: string | null, alt: string | null) {
-    return `<img src="${src}" alt="${alt ?? ""}" title="${title ?? ""}" />`;
+  image(src: string, title: string | null, alt: string) {
+    return `<img src="${src}" alt="${alt}" title="${title ?? ""}" />`;
```

Comment:
The "issue" with coverage that I previously asked about wasn't an issue -- we just weren't testing every branch!

In the course of trying to get 100% coverage, I found that it's impossible for `alt` to be null. If you cmd+click on the function, you'll see it's typed like this:
```ts
image(href: string, title: string | null, text: string): string;
```
so this just corrects our implementation.

Reply:
Good catch!